### DISCUSSION TITLE
How i add a new Disk(1:\)?

### DISCUSSION THREAD

**Discussion by FelekDevYT** [CosmosOS/Cosmos#3029] [General]
How i add a new Disk(1:\)?

**Comment by Gabolate** [comment]
You would have to add a new drive in your vm/pc so it appears in cosmos

**Reply by FelekDevYT** [reply]
ok , i test it

**Comment by ghost** [comment]
I think disk 1: is already occupied by the iso

**Reply by zarlo** [reply]
the iso will take the next free number so if 1 is in use it will take 2
